daemon/system-info.sh

```diff
@@ -393,6 +393,46 @@ elif pgrep "kubelet"; then
   HOST_IS_K8S_NODE="true"
 fi
 
+# ------------------------------------------------------------------------------------------------
+# Detect instance metadata for VMs running on cloud providers
+
+CLOUD_TYPE="unknown"
+CLOUD_INSTANCE_TYPE="unknown"
+CLOUD_INSTANCE_REGION="unknown"
+
+if [ "${VIRTUALIZATION}" != "none" ] && command -v curl > /dev/null 2>&1; then
```

Comment:
I tested on my host, and I observed an ancient problem when we reach this level, when I run on my host the script I have:

```sh
NETDATA_SYSTEM_VIRTUALIZATION=none
NETDATA_SYSTEM_VIRT_DETECTION=none
NETDATA_CLOUD_TYPE=unknown
NETDATA_CLOUD_INSTANCE_TYPE=unknown
NETDATA_CLOUD_INSTANCE_REGION=unknown
```

Considering that I am running direct on my host, we are missing something for our Virtualization detection. I am not sure if this is affecting all systems that do not use `systemd`.

Reply:
Sorry my fault!